misc/docs-linter/main_test.go

```diff
@@ -0,0 +1,125 @@
+package main
```

Comment:
All of these tests can be run in parallel

Reply:
[ca2a606](https://github.com/gnolang/gno/pull/2125/commits/ca2a606a2676293fcba80ad52ded81231dce9376)